nowcasting_dataset/config/model.py

```diff
@@ -64,6 +64,11 @@ class InputData(BaseModel):
         description="Path to the GeoTIFF Topographic data source",
     )
 
+    sun_filename: str = Field(
```

Comment:
Would it be OK to change `sun_filename` to something like `sun_zarr_path` to be consistent with the other Zarr paths?  I'm probably being overly pedantic (!) but, as I'm sure you've noticed, a Zarr dataset isn't a single file, it's a folder full of folders and files.

Reply:
yea - thats a good idea